### DISCUSSION TITLE
Change default editor when you "Open Locally"

### DISCUSSION THREAD

**Discussion by JGaylorCrestron** [Eugeny/tabby#9643] [Q&A]
I am running Tabby on a Mac (Sonoma 14.4.1), and when I SFTP into something and try to edit a file locally it always opens VS code on the windows side.  I faintly remember selecting this at some point, (I must have selected the wrong one).  I want to use VS code on the Mac side.

Can I change this somehow?

**Comment by Eugeny** [comment]
Editor choice is delegated to the OS, you can probably open "Get Info" for a file of that type and change it there
